### PR TITLE
updated path in notebook to 2023 aqcuired data

### PR DESCRIPTION
**Pull request by JKutt**
A quick PR to update the path to magnetic data file acquired 2023 for EOAS 350

### CODE REVIEW

**Comment by JKutt** [comment]
@santisoler , it looks like some of the notebook parameters changed when I saved to older versions of a few things. Will this matter. Should I maybe just edit the straight text?

**Comment by santisoler** [comment]
Thanks for this PR @JKutt! I suspect those values will be overwritten when the students open the notebook with the kernels in the JupyterHub. But if we want to be sure we are not going to break anything, we can totally restore the previous changes on those two lines. I can do it, no worries.

On a side note, I would merge this PR after the file is uploaded.

File: notebooks/mag/Mag_FitProfile.ipynb [hunk 1]
```diff
@@ -225,7 +225,7 @@
    }
   },
   "kernelspec": {
-   "display_name": "Python 3 (ipykernel)",
+   "display_name": "Python 3",
```
Comment (santisoler) [inline notebooks/mag/Mag_FitProfile.ipynb:228]:
```suggestion
   "display_name": "Python 3 (ipykernel)",
```

File: notebooks/mag/Mag_FitProfile.ipynb [hunk 2]
```diff
@@ -239,7 +239,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.10.4"
+   "version": "3.8.8"
```
Comment (santisoler) [inline notebooks/mag/Mag_FitProfile.ipynb:242]:
```suggestion
   "version": "3.10.4"
```

**Review by santisoler — CHANGES_REQUESTED** [review]
A few suggestions to restore the original kernel parameters in the notebook

**Review by santisoler — APPROVED** [review]

**Pull request merged**